src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -4,8 +4,9 @@ CHANGELOG
 6.3
 ---
 
- * Add `XmlEncoder::SAVE_OPTIONS` context option
- * Deprecate `MissingConstructorArgumentsException` in favor of `MissingConstructorArgumentException`
+* Add `PREVENT_NULLABLE_FALLBACK` context flag to `AbstractNormalizer` prevent nullable properties to be set to null when missing and not optional.
```

Comment:
please keep the space at the beginning

Reply:
Ah my bad, must have happened through sorting. Is fixed 🙂